eval/utils.py

```diff
@@ -275,6 +276,13 @@ def load_hf_tokenizer(
         token=os.getenv("HF_TOKEN", None),
     ):
         from transformers import AutoTokenizer
+
+        # Need to explicitly import the olmo tokenizer.
+        try:
```

Comment:
I don't really like having this here... is a bit annoying to have this warning if you don't care about olmo. Is there another way we can do this?

Reply:
Sure. I was imitating the warning in the [https://github.com/allenai/open-instruct/blob/main/open_instruct/finetune.py#L43](trainer) if OLMo isn't installed but we can do it another way.
- Option 1: Just try to important and fail silently.
- Option 2: Only important if `"olmo" in model_name_or_path`. Not perfect because it requires the user to include `olmo` in the model name, but that's already done [here](https://github.com/allenai/open-instruct/blob/main/scripts/submit_eval_jobs.py#L331) anyhow. We should probably add a note about this somewhere; can do that later.

Any preference?

Reply:
Yeah, I guess since I did it for finetuning its okay. Hopefully at some point we can remove these checks (maybe if/when olmo becomes a first-class HF model, instead of relying on importing code).

Reply:
OK, I'll just leave it. Annoying but not terrible.